src/views/List.jsx

```diff
@@ -86,7 +87,7 @@ export function List({ data, listToken }) {
 	}
 
 	return (
-		<>
+		<Box minHeight="100vh">
```

Comment:
Cool! So this wasn't a container issue in Layout.jsx? I was wondering why all the other pages were fine now and not this page.

Reply:
Aha! Nice fix, @jongranados!